docs/source/user_guide/model_understanding.ipynb

```diff
@@ -724,6 +724,45 @@
     "        print(\"Class:\", cls)\n",
     "        display(result[cls][\"plot\"])"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Find Confusion Matrix and thresholds for Binary Classification Pipelines\n",
+    "\n",
+    "For binary classification pipelines, EvalML also provides the ability to compare the actual positive and actual negative histograms, as well as obtaining the confusion matrices and ideal thresholds per objective."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.model_understanding import find_confusion_matrix_per_thresholds\n",
+    "\n",
+    "df, objective_thresholds = find_confusion_matrix_per_thresholds(pipeline_binary, X, y, n_bins=10)\n",
+    "df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "objective_thresholds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the above results, the first dataframe contains the histograms for the actual positive and negative classes, indicated by `pos_bins` and `neg_bins`. The `confusion_matrix` cell holds the confusion matrices for the associated threshold, and the `data_in_bins` holds a random subset of row indices (both postive and negative) that belong in each bin. The index of the dataframe represents the associated threshold. For instance, at index `0.1`, there is 1 positive and 309 negative rows that fall between `[0.0, 0.1]`.\n",
```

Comment:
nit-pick: This text is a little outdated now because the format of the dataframe changed slightly.